Documentation/pamphlet.md

```diff
@@ -0,0 +1,35 @@
+# WCROC Horticulture Display Garden Mobile Application and Administration Management
+
+### User interface
+This application was created to allow the visitors to engage with the garden, and leave feedback. Visitors can navigate the plants virtually and leave comments on the ones that are the most interesting to them. The application also allows for the user to like or dislike the flowers. 
+
+There are three ways that a visitor can get to a specific flower page: 
+* Bed's QR code
+* Search by common name
+* Use bed's dropdown menu
+
+There is a 'Find a flower' search bar that lets the user search for plants by their _Common name_ which are then displayed in order of beds.  
+
+Once the visitor is in a specific flower page, they can then rate the flower by liking or disliking it, and/or leave a comment. 
+
+
+
+The app is meant to help Steve and Esther gather information on preferences and suggestions about the garden. At the bottom of all the pages in the application, there are links to social media and to the WCROC official website.
+
+
+### Administration side
+The admin side has features that only authorized WCROC staff can utilize. 
+<br>These features include: 
+* Import a spreadsheet with appropriate data (.xlsx)
+    * Updating data 
```

Comment:
I thought that the flower data could be update rather than deleting it and replacing it. For example if they received a new flower, couldn't they just 'update' and it would just add this flower to the data without deleting any of the current data?